### ISSUE TITLE
Modify relevant parameters to increase model performance

### ISSUE THREAD

**Issue by Aleksandretta** [FindMyKorki/sentiment-analysis#12]
Try modifying these values:

- learning rate
- patience
- max_seq_len (>50)

**Comment by izabelakolodziejska** [comment]
I did several tries to modify parameters, but unfortunately changes in performance isn't significant. Still i think number 8) might be worth consideration as it has 78% accuracy (in final accuracy print, in some epochs it had 79% even) 

1) starting base
max_seq_len = 50
optimizer = 2e-5
patience = 7

result 77% early stopping 89/100

2)
max_seq_len = 75
optimizer = 2e-5
patience = 7

result 77% early stopping 76/100

3) 
max_seq_len = 85
optimizer = 2e-5
patience = 7

result 76% early stopping 92/100

4) 
max_seq_len = 75
optimizer = 5e-5
patience = 7

result 75% early stopping 55/100

5)
max_seq_len = 75
optimizer = 3e-5
patience = 7

result 78% early stopping 67/100

6)
max_seq_len = 75
optimizer = 3e-5
patience = 10

result 76% early stopping 79/100

7)
max_seq_len = 75
optimizer = 3e-5
patience = 15

result 76% early stopping 84/100

8)
max_seq_len = 75
optimizer = 3e-5
patience = 5

result 78% early stopping 58/100

9)
max_seq_len = 90
optimizer = 3e-5
patience = 7

result 77% early stopping 60/100

10)
max_seq_len = 75
optimizer = 1e-5
patience = 7

result 78% no early stopping

**Comment by Aleksandretta** [comment]
Thank you for the good work! 
In case of our model and because we don't have that much training data, I think **option 5** seems to be the best. Greater patience may be more beneficial as it allows the model to train longer and take better guesses about the weights. Though, we do that with cost of higher early stopping percentage which causes longer trainig, but again our (own) data is limited, so I think that allowing the model to have more time to optimize the weights outweighs the cons.
